### PR TITLE
Add 'j-jobs' for UFS DA

### LINKED ISSUE #7: Add 'j-jobs' for UFS DA
Need three 'j-jobs' for each new task for UFS DA:

- [ ] UFS DA atmos analysis prep
- [ ] UFS DA atmos analysis run
- [ ] UFS DA atmos analysis post

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
Closes #7 

Adds three `j-jobs` for UFS DA: prep, run, post.

- prep: calls a python ex-script to create YAML and stage obs, backgrounds, fix files, etc
- run: calls a shell script to run fv3jedi_var.x and then update analysis fields
- post: calls a python script to concatenate diag files and save them in R2D2;

### CODE REVIEW

**Review by RussTreadon-NOAA — COMMENTED** [review]
Two general comments:

1. global-workflow PR [#526](https://github.com/NOAA-EMC/global-workflow/pull/526) replaced all backticks for command substitution.   As we develop new UFS DA j-jobs we should follow suit.  There may be other standards EIB / NCO would like us to follow.   It's best to start off new jobs with these standards in mind.
2. We often see $base_init, $base_run, $base_finalize as a naming subroutines.   The UFS DA j-jobs in this PR reverse this pattern.  We have init_$base, run_$base, and finalize_$base.   My preference is the former instead of the latter.   This is only a comment & not a change request.

**Comment by CoryMartin-NOAA** [comment]
Thanks @RussTreadon-NOAA , are the job names more what you were looking for? I agree the backticks should be replaced to be consistent, and I think that has been now fixed too.

**Review by RussTreadon-NOAA — DISMISSED** [review]
Thank you, @CoryMartin-NOAA , for renaming the jobs.  Yes, this is what I was thinking of.   I don't think everything else is in place to fully test these jobs so I'll approve based on what I see.  This is getting all the wiring in place.  We may need to tweak things after we turn on the power.

**Comment by CoryMartin-NOAA** [comment]
after talking with @aerorahul , I have consolidated these into just one j-job, and the unique things will be passed via environment variables (configs and ex-script path)

File: UFS-DA/jobs/JUFSDA_GLOBAL_ATMOS_ANALYSIS [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+export COMPONENT=${COMPONENT:-atmos}
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+GDATE=$($NDATE -$assim_freq $CDATE)
```
Comment (aerorahul) [inline UFS-DA/jobs/JUFSDA_GLOBAL_ATMOS_ANALYSIS]:
```suggestion
GDATE=$(date +%Y%m%d%H -d "${CDATE:0:8} ${CDATE:8:2} - ${assim_freq} hours")
```

File: UFS-DA/jobs/JUFSDA_GLOBAL_ATMOS_ANALYSIS [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+export OPREFIX="${CDUMP}.t${cyc}z."
+export GPREFIX="${GDUMP}.t${gcyc}z."
+export APREFIX="${CDUMP}.t${cyc}z."
+export GSUFFIX=${GSUFFIX:-$SUFFIX}
+export ASUFFIX=${ASUFFIX:-$SUFFIX}
+
+export COMOUT=${COMOUT:-$ROTDIR/$CDUMP.$PDY/$cyc/$COMPONENT}
+
+mkdir -m 775 -p $COMOUT
```
Comment (aerorahul) [inline UFS-DA/jobs/JUFSDA_GLOBAL_ATMOS_ANALYSIS]:
```suggestion
mkdir -p $COMOUT
```

**Review by aerorahul — APPROVED** [review]
changes look good.

**Review by CatherineThomas-NOAA — APPROVED** [review]

**Pull request merged**